regression/tests/studio/test_schedule_and_details_settings.py

```diff
@@ -94,3 +102,70 @@ def test_cancel_upload(self):
             self.settings_page.get_element(
                 '.wrapper-input input'
             ).get_attribute('value'))
+
+
+class ScheduleAndDetailsLinks(WebAppTest):
+    """
+    Tests for Studio's Setting page links.
+    """
+    def setUp(self):
+        super(ScheduleAndDetailsLinks, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.course_info = get_course_info()
+
+        self.settings_page = SettingsPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        self.settings_page.visit()
+
+    def test_other_grading_link(self):
+        """
+        Verifies that user can click and navigate to Grading
+        """
+        link = 'href="/settings/grading/course-v1:ArbiRaees+AR-1000+fall"'
+        grading_page = GradingPageExtended(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run'])
+        self.settings_page.click_other_settings_links(link)
+        grading_page.wait_for_page()
+
+    def test_other_course_team_link(self):
+        """
+        Verifies that user can click and navigate to Course Team
+        """
+        link = 'href="/course_team/course-v1:ArbiRaees+AR-1000+fall"'
```

Comment:
At the very least, since it's the same course, you should have this as a var on the top of the file.